src/apis/fetchSubmitLogin.ts

```diff
@@ -0,0 +1,15 @@
+import { userInformationProps } from "../types";
+
+export const fetchSubmitLogin = async (userInformation: userInformationProps): Promise<boolean> => {
+    let loggedIn: boolean = false;
+    const password = userInformation.password;
+    const userName = userInformation.userName;
+    const url = `http://localhost:3001/users?username=${userName}&password=${password}`;
+    let fetchResult = await fetch(url);
+    let response = await fetchResult.json().then((data) => {
```

Comment:
its better to just return the json response here and return another operation to the callee  this will e much useful ,especially with  apis like product / categories that could be called from many places for multi purposes